helmfile/values/userCRDs/bitnami/sealedsecrets.yaml.gotmpl

```diff
@@ -0,0 +1,98 @@
+{{- if .Values.gatekeeper.sealedSecrets.enabled }}
+userCRDs:
+    allowedCRDs:
+    - names:
+        - sealedsecrets.bitnami.com
+      group: "bitnami.com"
```

Comment:
I might be missing something, but it does not look like this is being loaded into the gatekeeper constraints chart and it does not seem to be needed in the user-crds chart where it is loaded. Same for the mongodb values.